### ISSUE TITLE
Should OGC API standards offer Static Web Server capabilities?

### ISSUE THREAD

**Issue by ogcscotts** [ogcscotts/TC-Meeting-topics#167]

**Comment by cnreediii** [comment]
Yes. If OGC APIs are to work with data stores such as defined by the CDB standard, than static servers need to be supported. Also, I looked up what the definition of a static web server is. From the Mozilla website:

A static web server, or stack, consists of a computer (hardware) with an HTTP server (software). We call it "static" because the server sends its hosted files as-is to your browser. A dynamic web server consists of a static web server plus extra software, most commonly an application server and a database.
